### PR TITLE
docs: updated readme and setup guide

### PR DESCRIPTION
**Pull request by crisgarner**
- Updated Readme
- Update sample.env
- Updated setup with a guide to run MACI

### CODE REVIEW

**Review by ctrlc03 — CHANGES_REQUESTED** [review]
Thanks, left some comments

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 ## Supported Networks
 
-All networks EAS is deployed to are supported
+All networks EAS is deployed to are supported. If the network is not supported, you can follow EAS documentation to deploy the contracts in the network.
```
[the remark this answers is not in the document]
Comment (kittybest) [reply README.md:31]: is that means that the user could deploy EAS contract on a different network? is there any related documentation? maybe could put it here.

File: docs/01_setup.md [hunk 2]
```diff
@@ -58,9 +58,86 @@ To create your own round you need to do a few things:
...
+```bash
+node build/ts/index.js genMaciKeyPair
+```
+
+### Deploy MACI Contracts
+
+The following scripts deploy a MACI instance and set the keys, make sure to save the address of the MACI contract.
+
+> [!IMPORTANT]
+> Make sure that you use the NonQv keys
```
Comment (kittybest) [inline docs/01_setup.md:110]: question: they could only use nonQV version, or actually they can choose to deploy QV or nonQV one?

**Comment by kittybest** [comment]
open an issue for this: #83

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (crisgarner) [reply README.md:31]: > is that means that the user could deploy EAS contract on a different network? is there any related documentation? maybe could put it here.

yes! you can do a deploy with their docs here https://docs.attest.org/docs/quick--start/contracts#deploying

[the remark this answers is not in the document]
Comment (kittybest) [reply README.md:31]: Then I think we could add this link to the `README.md` file?

[the remark this answers is not in the document]
Comment (crisgarner) [reply README.md:31]: It's there, on the new PR. 

```All networks EAS is deployed to are supported. If the network is not supported, you can follow EAS documentation to deploy the contracts in the network.

- https://docs.attest.sh/docs/quick--start/contracts```

File: docs/01_setup.md (hunk 2 above)
Comment (crisgarner) [reply docs/01_setup.md:110]: Maybe we should try to run a QV round to see what happens

File: README.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+</div>
 
-<div>
-
-[<img src="./docs/images/screenshot.png">](https://easy-retro-pgf.vercel.app)
+[<img src="./docs/images/screenshot.png"/>](https://easy-retro-pgf.vercel.app)
 
 ## Documentation
 
+MACI-RPGF uses EAS as backbone to run Retroactive Public Goods Funding to reward contributors ([As used by the Optimism Collective](https://community.optimism.io/docs/governance/citizens-house/#how-retro-funding-works)) while adding a private layer to reduce bribery and collusion using MACI.
```
Comment (ctrlc03) [inline README.md:13]: privacy layer maybe?

File: README.md [hunk 4]
```diff
@@ -28,7 +28,7 @@
 
 ## Supported Networks
 
-All networks EAS is deployed to are supported
+All networks EAS is deployed to are supported. If the network is not supported, you can follow the EAS documentation to deploy the contracts to the network.
```
Comment (ctrlc03) [inline README.md:31]: nit: if a network is not supported [..] deploy the contracts to that network.

File: docs/01_setup.md [hunk 5]
```diff
@@ -58,9 +58,85 @@ To create your own round you need to do a few things:
 
 If you are running on a different network than Optimism you need to update the contract addresses for EAS. These addresses are used whenever an attestation is created.
 
-You can also configure your own schemas here if you wish to.
+You can also configure your own schemas here if you wish to, or deploy the EAS contracts to a network that doesn't have it.
```
Comment (ctrlc03) [inline docs/01_setup.md:61]: them not it

**Review by ctrlc03 — CHANGES_REQUESTED** [review]
Thanks, left couple more comments

**Comment by crisgarner** [comment]
fixed the comments @ctrlc03 🫡

**Review by kittybest — APPROVED** [review]
thanks!

**Review by ctrlc03 — APPROVED** [review]
Thanks

**Pull request merged**